client/components/logged-out-form/test/link-item.js

```diff
@@ -1,21 +1,25 @@
-import { shallow } from 'enzyme';
+/**
+ * @jest-environment jsdom
+ */
+
+import { render, screen } from '@testing-library/react';
 import LoggedOutFormLinkItem from '../link-item';
 
 describe( 'LoggedOutFormLinkItem', () => {
 	test( 'should render an <a> element', () => {
-		const wrapper = shallow(
+		render(
 			<LoggedOutFormLinkItem href="http://example.com">Example text here</LoggedOutFormLinkItem>
 		);
+		const link = screen.queryByRole( 'link' );
 
-		expect( wrapper ).toMatchSnapshot();
```

Comment:
👍 This definitely looked like one of those snapshots that doesn't add much value, so thanks for removing it.